docs/GettingStarted.md

```diff
@@ -3,47 +3,252 @@
+
+``` bash
+az group create --name ${RG_TELEMETRYPLATFORM} --location ${LOCATION}
+```
+
+- Execute the main.bicep targeting your telemetry platform resource group
+
+``` bash
+az deployment group create --resource-group ${RG_TELEMETRYPLATFORM}--template-file ./main.bicep 
+```
+
+#### Deploy the telemetry platform functions
+
+* Change directory to ```./src/TelemetryPlatform/Functions```
+
+* Export a variable with the name of the function app created in the telemetry platform
+
+```bash
+export tpfunctionapp=$(az functionapp list --query "[].name" --resource-group ${RG_TELEMETRYPLATFORM} --output tsv)
+```
+
+* Publish the functions to the function app
+
+```bash
+func azure functionapp publish ${tpfunctionapp} --dotnet
+```
+
+### Deploy the fleet integration layer
+
+In this step you will deploy the resources required for the fleet integration layer
+
+#### Execute fleet integration biceps deployment scripts
+
+- Change your directory to ```infra/deployment/FleetIntegration```
 
-- Follow the instructions to deploy the infrastructure for each layer.
-  - [Telemetry Platform](../infra/deployment/TelemetryPlatform/README.md) requires creating test certificates andcreating Azure resources for Event Grid, Event Grid Topics, Azure Functions and test devices.
-  - [Fleet Integration](../infra/deployment/FleetIntegration/README.md) deploy the resources to process messages into analytics and business integration.
+- [Create a resource group](https://learn.microsoft.com/cli/azure/manage-azure-groups-azure-cli#create-a-resource-group) for the deployment in a region
 
-- Deploy and configure the Azure Functions
-  - [Deploy and configure Telemetry Platform](../src/TelemetryPlatform/Functions/README.md) deploys and configures the message processing code
-  - [Deploy and configure Fleet Integration](../src/FleetIntegration/Functions/README.md) deploys and configures the dataverse integration code
+``` bash
+az group create --name ${RG_FLEETINTEGRATION} --location ${LOCATION}
+```
+
+- Get the name of the event hub created in the telemetry platform.
+
+``` bash
+export tpeventhubname=$(az eventhubs namespace list --resource-group ${RG_TELEMETRYPLATFORM} --query "[].name" --output tsv)
+```
+
+- Execute the main.bicep refering to your resource group and using the event hub namespace from the telemetry platform as an argument.
+
+``` bash
+az deployment group create --resource-group ${RG_FLEETINTEGRATION} --template-file ./main.bicep --parameters evhnsTelemetryPlatformNamespaceName=${tpeventhubname}
+```
+
+You can try out sending messages and the integration with Azure Data Explorer.
+
+#### Deploy the telemetry platform functions for connection to the dataverse
+
+> [!NOTE]
+> This step requires a Dynamics 365 installation with Field Service. You can skip this step in case you don't have an installation available.
```

Comment:
Thank you for splitting this out! It makes it a lot easier to follow along when you know it's an optional step that won't block the rest of the sample.